lib/product_detail/apis/product_api.dart

```diff
@@ -1,18 +1,27 @@
 import 'package:dio/dio.dart';
+import 'package:flex_storefront/config/config_repository.dart';
+import 'package:flex_storefront/config/data/default_config.dart';
 import 'package:flex_storefront/init.dart';
 import 'package:flex_storefront/product_list/models/product.dart';
 import 'package:flutter_dotenv/flutter_dotenv.dart';
 import 'package:get_it/get_it.dart';
 
-const PATH = '/occ/v2/electronics-spa/products/';
+const PATH = '/occ/v2/<CATALOG>/products';
 const PARAMS =
     '?fields=code,configurable,configuratorType,name,summary,price(formattedValue,DEFAULT),images(galleryIndex,FULL),baseProduct,averageRating,stock(DEFAULT),description,availableForPickup,url,numberOfReviews,manufacturer,categories(FULL),priceRange,multidimensional,tags&lang=en&curr=USD';
 
 class ProductApi {
   Future<Product> fetchProduct({required String productId}) async {
+    final path = PATH.replaceAll(
```

Comment:
Hmm maybe we move these paths to a utility class. I have an idea, we can discuss next meeting.